views/home/faq.html

```diff
@@ -14,75 +14,105 @@
+            Please make sure you have read the <a href="/tos">Terms of Service</a>.
+           </p>'
+    },{
+    'id': 'gui',
+    'q':  'Can I get a GUI session?',
+    'a':  'Click the <button type="button" class="btn btn-blue btn-xs">
+           <img src="/assets/images/icon-display.png" style="width:14px;margin-top:-2px" /></button>
+           button to get an interactive GUI session inside your browser.'
+    },{
+    'id': 'projects',
+    'q':  'What are projects?',
+    'a':  'Projects are groups that work together, every member of a project will be able to see all VMs in said project.
+           Use the drop-down in the top right of the page to switch between projects.',
+    'p':  'openstack'
+    },{
+    'id': 'newprojects',
+    'q':  'Can I create/join another project?',
+    'a':  'Yes, if you would like to create/join another project email us at
+           <a href="mailto:' + email + '">' + email + '</a>.',
+    'p':  'openstack'
+    },{
+    'id': 'rename',
+    'q':  'Can I rename my VM?',
+    'a':  'Yes, simply mouseover the VM name and click the 
+           <span class=" btn btn-default btn-xs glyphicon glyphicon-pencil" style="vertical-align:middle;margin-top:-2px"></span>
+           button that appears. It may take a few moments for the name to update on the machines page.',
+    'p':  'openstack'
+    },{
+    'id': 'flavors',
+    'q':  'What are flavors?',
+    'a':  'Flavors define the resources assigned to a VM when it is created.
+           Once you have chosen a template, you will be prompted to choose a flavor from the drop-down.',
+    'p':  'openstack'
+    },{
+    'id': 'resources',
+    'q':  'Can I increase the CPUs, RAM or Disk size?',
+    'a':  'Currently we only provide a standard set of resources for each machine.
+           If you would like to discuss increasing these values email us at
+           <a href="mailto:' + email + '">' + email + '</a>.',
+    },{
+    'id': 'quota',
+    'q':  'Can I increase my quota?',
+    'a':  'Yes, if you would like your quota increased email us at
+           <a href="mailto:' + email + '">' + email + '</a>.'
+    },{
+    'id': 'stopping',
+    'q':  'What is the <span class="label label-warning">STOPPING</span> state?',
+    'a':  'When a VM is deleted, a snapshop is taken an stored for a period of time.
+           This is for the reason of security and traceability'
+    },{
+    'id': 'os',
+    'q':  'Do you support other Linux variants/version?',
+    'a':  'Yes, if you would like to discuss adding different images or versions to the cloud, email us at 
+           <a href="mailto:' + email + '">' + email + '</a>.',
+    },{
+    'id': 'pakiti',
+    'q':  'Why are Pakiti and rsyslog running on my VM?',
+    'a':  'These services are used for monitoring and security.
+           Please refrain from stopping or changing the behaviour of either of these service.
```

Comment:
Should probably use a phrase that is stronger than just "refrain".

Maybe: "You must not stop or change the behaviour of..."